src/backend/nodes/cypher_readfuncs.c

```diff
@@ -268,9 +268,9 @@ void read_cypher_delete_information(struct ExtensibleNode *node)
     READ_LOCALS(cypher_delete_information);
 
     READ_NODE_FIELD(delete_items);
-    READ_UINT_FIELD(flags);
+    READ_INT_FIELD(flags);
     READ_STRING_FIELD(graph_name);
-    READ_UINT_FIELD(graph_oid);
+    READ_INT_FIELD(graph_oid);
```

Comment:
Why were these changed back to INT from UINT. They were purposely changed to UINT in the master.